tests/steps/other_v2_steps.py

```diff
@@ -1441,9 +1442,12 @@ def simulate_transaction(context):
 @then("the simulation should succeed without any failure message")
 def simulate_transaction_succeed(context):
     if hasattr(context, "simulate_response"):
-        assert context.simulate_response["would-succeed"] is True
+        assert len(context.simulate_response["failure-message"]) == 0
```

Comment:
Is this right? I would think you'd need to index into `txn-groups` to get at this

Reply:
no, this is def not right, I am mucking around with changed response, will update later.